src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const value of actions) {
```

Comment:
come on, [checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) 3rd item, use proper variable names when iterating through array using `for ... of`
```suggestion
  for (const action of actions) {
```